src/Build/Evaluation/Context/EvaluationContext.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 using System.Collections.Concurrent;
-using System.Collections.Immutable;
+using System.Collections.Generic;
```

Comment:
I am switching over from `ImmutableArray<string>` to `IReadOnlyList<string>` because the former is ~4x more expensive to construct from an `IEnumerable`. We don't really use the immutable functionality here as we never "mutate" the collection after it's created.